src/components/Footer/LinksList.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from "react";
+import styled from "styled-components";
+
+import { Icon } from "../Icon";
+import { Text } from "../Text";
+import { Title } from "../Title";
+
+interface LinkListElement {
+  /**
+   * Link text
+   */
+  label: string;
+  /**
+   * External link element
+   */
+  linkComponent?: JSX.Element;
+}
+
+export interface LinksListProps {
+  title?: string;
+  links?: LinkListElement[];
+}
+
+const LinksListRoot = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-transform: uppercase;
+`;
+
+const LinkElementWrapper = styled.div`
+  position: relative;
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 4px 0px;
```

Comment:
Ho visto che nell'invision i link sono in una box unica che ha come stile:

```
font-size: 16px;
line-height: 34px;
```
Quindi credo che il margin dovrebbe essere di 5px (ossia: (34px - 24px)/2) oppure puoi cambiare la line-height